src/utils/helpers.utils.js

```diff
@@ -74,9 +74,54 @@ export const getArticlebyId = async id => {
  * @returns {Boolean} false if Article does not exist
  */
 export const getArticlebySlug = async slug => {
-  return await Article.findOne({ where: { slug } });
+  return await Article.findOne({
+    where: { slug },
+    include: {
+      model: User,
+      as: 'author',
+    },
+  });
 };
 
+// /**
+//  * Check User existence
+//  *
+//  * @param {integer} id
+//  * @returns {Boolean} true if username exists
+//  * @returns {Boolean} false if username does not exist
+//  */
+// export const getUserbyId = async id => {
+//   return await User.findByPk(id);
+// };
+
+// /**
+//  * Check Article existence
+//  *
+//  * @param {String} id
+//  * @returns {Boolean} true if Article exists
+//  * @returns {Boolean} false if Article does not exist
+//  */
+// export const getArticlebyId = async id => {
+//   return await Article.findOne({
+//     where: { id },
+//     include: {
+//       model: User,
+//       as: 'user',
+//     },
+//   });
+// };
+
+// /**
+//  * Check Article existence
+//  *
+//  * @param {String} slug
+//  * @returns {Boolean} true if Article exists
+//  * @returns {Boolean} false if Article does not exist
+//  */
+// export const getArticlebySlug = async slug => {
+//   return await Article.findOne({ where: { slug } });
+// };
+
```

Comment:
Hi, Nice work. but please can you remove this commented out code, or uncomment it.

Reply:
done